string plant_1
string plant_2
string plant_3
string plant_4
string plant_5
string plant_6